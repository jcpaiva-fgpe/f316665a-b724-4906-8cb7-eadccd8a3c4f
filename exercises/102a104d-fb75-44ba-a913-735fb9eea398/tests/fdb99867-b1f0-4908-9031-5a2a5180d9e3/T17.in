9{40[60{25[h]}]}+[31[77[j]]]-51[([29[{{s}-79w}]])+2(67x-{[68([83w-x])+{([1s])-98{69{23p}}-22p}]+{e+65{47[(24y)]-x}-11(83{90i})}+({e+92a})}-{81(90y+[z]-10[99(l)]-46{20{(33d)}})}+([c-43{46a+27[j]+87s-d}]))]-g
s+({55[0f]+[13m+h+[y]]}-2
