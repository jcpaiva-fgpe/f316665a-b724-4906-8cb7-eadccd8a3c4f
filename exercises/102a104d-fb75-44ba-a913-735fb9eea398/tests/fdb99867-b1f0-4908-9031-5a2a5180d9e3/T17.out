[] 10 12
{} 7 13
[] 4 14
{} 1 15
[] 23 25
[] 20 26
[] 17 27
{} 38 40
{} 37 45
[] 36 46
[] 33 47
() 32 48
[] 61 67
() 60 68
[] 72 75
() 71 76
{} 83 87
{} 80 88
{} 70 93
[] 57 94
() 105 109
[] 104 110
{} 101 113
{} 120 124
() 117 125
{} 96 126
{} 129 135
() 128 136
{} 56 137
[] 147 149
() 156 158
[] 153 159
() 167 171
{} 166 172
{} 163 173
() 142 174
{} 139 175
[] 190 192
{} 183 199
[] 178 200
() 177 201
() 51 202
[] 31 203
